doc/esmvalcore/preprocessor.rst

```diff
@@ -111,9 +111,55 @@ See also :func:`esmvalcore.preprocessor.derive`.
 
 CMORization and dataset-specific fixes
 ======================================
-.. warning::
 
-   Section to be added.
+Data checking
+-------------
+
+Data preprocessed by ESMValTool is automatically checked against its cmor
+definition. To reduce the impact of this check while maintaing it as realiable
+as possible, it is splitted in two parts: one will check the metadata and will
+be done just after loading and concatenating the data and the other one will
+check the data itself and will be applied after all extracting operations are
+applied to reduce the amount of data to process.
+
+Checks include, but are not limited to:
+
+   - Requested coordinates are present and comply with their definition.
+   - Correctness of variable names, units and other metadata.
+   - Compliance with the valid minimum and maximum values allowed if defined.
+
+The most relevant (i.e. a missing coordinate) will raise an error while others
+(i.e an incorrect long name) will be reported as a warning.
+
+Somo of those issues will be fixed automatically by the tool, including the
+following:
+
+    - Incorrect standard or long names.
+    - Incorrect units, if they can be converted to the correct ones.
+    - Direction of coordinates.
+    - Automatic clipping of longitude to 0 - 360 interval.
+
+
+Dataset specific fixes
+----------------------
+
+Sometimes, the checker will detect errors that it can not fix by itself.
+ESMValTool deals with those issues by applying specific fixes for those
+dataset that require them. Fixes are applied at three different preprocessor
+steps:
+
+    - fix_file: apply fixes directly to a copy of the file. Only fixes that allow
```

Comment:
change to something like: 'Only fixes that prevent iris from loading the file are applied here.'